testing-standard.md

```diff
@@ -40,7 +41,7 @@ expect(myStub).to.have.been.called();
 
 
 ## 1.2 Good Test Folders
-When addind new tests to projects, test files should sit in a folder hierarchy that mimics the hierarchy fo the source code. The test files themselves should be named similarly to the source files, but with `test` at the beginning of the file name. For example, if we have a source file `src/views/checkoutView.js` we should have a test file `test/unit/views/checkoutView.js` or similar. This makes finding tests an easy and logical process.
+When adding new tests to projects, test files should sit in a folder hierarchy that mimics the hierarchy fo the source code. The test files themselves should be named similarly to the source files, but with `test` at the beginning of the file name. For example, if we have a source file `src/views/checkoutView.js` we should have a test file `test/unit/views/checkoutView.js` or similar. This makes finding tests an easy and logical process.
```

Comment:
The test prefix allows us to quickly search for test files, it's easy to see it's a test when the filename is really long for example. Plus it keeps the filenames in camelcase format.
```
test/unit/views/checkoutViewReallyReallyLongFileName_test.js
test/unit/views/testCheckoutViewReallyReallyLongFileName.js
```